app/scripts/badges.js

```diff
@@ -12,14 +12,14 @@ export default () => [
 
 
 '<div style="width: 100px; height: 100px; display: inline; padding-right: 5px" id="my-badge" class="tooltip">',
-'<span class="tooltiptext">Percentage of Repository Activity</span>',
+'<span class="tooltiptext">Percentage calculated based on the number of commits, the number of contributors, accepted PRs, and active issues of the last month. Also, if release notes exist.</span>',
 '</div>',
 
 '<div style="width: 100px; height: 100px; display: inline; padding-right: 5px" id="my-badge2" class="tooltip">',
-'<span class="tooltiptext">Percentage of User Support</span>',
+'<span class="tooltiptext">Percentage calculated based on the checklist of the community profile, the number of contributors and accepted PRs and also based on tags, features, and labels for PRs and issues.</span>',
```

Comment:
> and also based on tags, features, and labels for PRs and issues.

What are those tags, features, and labels?

Reply:
labels as "good first issue" and "help wanted", for example.